### ISSUE TITLE
Confusing installation instructions

### ISSUE THREAD

**Issue by slothstronaut** [getsentry/sentry-wizard#71]
The looping gif video is not helpful. It should be a video that you can pause to see what it's doing, or nothing at all.

The instructions "call sentry-wizard" also are vague, what it _actually_ means is `yarn sentry-wizard`, not everyone is going to just know that call == `yarn`, would be better to just specify what command to run exactly instead of leaving people guessing.

Also the gif shows the command `yarn try` which doesn't seem to do anything.

**Comment by kettunen** [comment]
Yeah, bad instructions. Couldn't install this to Angular project with npm while there's link to this wizard in https://docs.sentry.io/platforms/javascript/guides/angular/sourcemaps/uploading/

**Comment by leojh** [comment]
terrible
